revolv/templates/project/project.html

```diff
@@ -91,4 +93,133 @@
     }
     google.maps.event.addDomListener(window, 'load', initialize);
 </script>
+
+<script type="text/javascript">
+$(function() {
+    // http://stackoverflow.com/questions/610406
+    if (!String.prototype.format) {
+        String.prototype.format = function() {
+            var args = arguments;
+            return this.replace(/{(\d+)}/g, function(match, number) { 
+                return typeof args[number] != 'undefined' ? args[number] : match;
+            });
+        };
+    }
+
+    var prettyName = function(name) {
+        return name.replace(/_/g, ' ')
+                .replace(/\w\S*/g, function(txt) {
+                    return txt.charAt(0).toUpperCase() + txt.substr(1).toLowerCase();
+                });
+    };
+
+    var cardInfoBlock = function(info) {
+        // TODO: style block
+        var block = $('<ul/>').addClass('confirm-info-card');
+        block.append($('<li/>')
+            .text('{0} ending in {1}'.format(
+                prettyName(info.type), info.number.substr(-4)
+            )));
+        block.append($('<li/>')
+            .text('Expires: {0}/{1}'.format(
+                info.expire_month, info.expire_year
+            )));
+        block.append($('<li/>')
+            .text('Name on card: {0} {1}'.format(
+                info.first_name, info.last_name
+            )));
+        return block;
+    };
+
+    var ffNames = {{ form.fields.keys | safe }}; // TODO: probably not okay, #yolo
+    $('#donate-form').submit(function(e) {
+        e.preventDefault();
+        var form = $(this);
+        var formURL = form.attr('action');
+        var formData = form.serializeArray();
+        $.ajax({
+            url: '{{ request.get_full_path }}' + formURL,
+            data: formData,
+            type: 'POST',
+            success: function(data) {
```

Comment:
Maybe it's can be more concise to use `$.post`? And people tend to like using promises instead of the success function nowadays - it wouldn't change much though. http://jqfundamentals.com/chapter/ajax-deferreds